clang/test/CodeGenSYCL/debug-info-srcpos-kernel.cpp

```diff
@@ -38,3 +38,13 @@ int main() {
 // CHECK: [[LINE_B0]] = !DILocation(line: 16,{{.*}}scope: [[BLOCK:![0-9]+]]
 // CHECK: [[BLOCK]] = distinct !DILexicalBlock(scope: [[KERNEL]]
 // CHECK: [[LINE_C0]] = !DILocation(line: 17,{{.*}}scope: [[KERNEL]]
+
+// RUN: %clang -fno-sycl-force-inline-kernel-lambda -fsycl-device-only %s -S -emit-llvm -O0 -g -o - | FileCheck %s
+//
+// Verify the SYCL kernel routine is marked artificial and has the
+// expected source correlation.
+//
+// In order to placate the profiling tools, which can't cope with instructions
+// mapped to line 0, we've made the change so that the artificial code in a
+// SYCL kernel gets the source line info for the kernel caller function (the
+// 'kernel' template function on line 15 in this file).
```

Comment:
That looks like a pure copy-paste of the beginning of this test. Could you please remove or explain what does it try to achieve?

Reply:
Yeah, this is a mistake.